test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Services/NuGetPackageSearchServiceTests.cs

```diff
@@ -155,7 +162,7 @@ public async Task GetTotalCountAsync_WithLessThanMaxCountResults_ReturnsResultsC
                     ItemFilter.All,
                     CancellationToken.None);
 
-                Assert.Equal(22, totalCount);
+                Assert.Equal(1, totalCount);
```

Comment:
Can you elaborate on this change?  I don't understand it.

Reply:
I changed the testdata, the result data set only contains one item now.